pkg/gui/controllers/local_commits_controller.go

```diff
@@ -816,25 +815,62 @@ func (self *LocalCommitsController) createFixupCommit(commit *models.Commit) err
 	})
 }
 
-func (self *LocalCommitsController) squashAllAboveFixupCommits(commit *models.Commit) error {
-	prompt := utils.ResolvePlaceholderString(
-		self.c.Tr.SureSquashAboveCommits,
-		map[string]string{"commit": commit.Sha},
-	)
-
-	return self.c.Confirm(types.ConfirmOpts{
-		Title:  self.c.Tr.SquashAboveCommits,
-		Prompt: prompt,
-		HandleConfirm: func() error {
-			return self.c.WithWaitingStatus(self.c.Tr.SquashingStatus, func(gocui.Task) error {
-				self.c.LogAction(self.c.Tr.Actions.SquashAllAboveFixupCommits)
-				err := self.c.Git().Rebase.SquashAllAboveFixupCommits(commit)
-				return self.c.Helpers().MergeAndRebase.CheckMergeOrRebase(err)
-			})
+func (self *LocalCommitsController) squashFixupCommits() error {
+	return self.c.Menu(types.CreateMenuOptions{
+		Title: self.c.Tr.SquashAboveCommits,
+		Items: []*types.MenuItem{
+			{
+				Label:          self.c.Tr.SquashCommitsInCurrentBranch,
+				OnPress:        self.squashAllFixupsInCurrentBranch,
+				DisabledReason: self.canFindCommitForSquashFixupsInCurrentBranch(),
+				Key:            'b',
```

Comment:
I was also unsure about the keybindings for the menu entries, especially this one. Any thoughts?

Reply:
I'm fine with it